lib/project_types/rails/messages/messages.rb

```diff
@@ -26,15 +26,15 @@ module Messages
 
         create: {
           help: <<~HELP,
-          {{command:%s create rails}}: Creates a ruby on rails app.
+          {{command:%s create rails}}: Creates a Ruby on Rails app.
             Usage: {{command:%s create rails}}
             Options:
-              {{command:--name=NAME}} App name. Any string.
-              {{command:--app-url=APPURL}} App URL. Must be a valid URL.
-              {{command:--organization-id=ID}} Partner organization ID. Must be an existing organization.
-              {{command:--shop-domain=MYSHOPIFYDOMAIN }} Development store URL. Must be an existing development store.
-              {{command:--db=DB}} Database type. Must be one of: mysql, postgresql, sqlite3, oracle, frontbase, ibm_db, sqlserver, jdbcmysql, jdbcsqlite3, jdbcpostgresql, jdbc.
-              {{command:--rails-opts=RAILSOPTS}} Additional options. Must be string containing one or more valid Rails options, separated by spaces.
+              {{command:--name=<name>}}           App name. Any string.
+              {{command:--app-url=<url>}}         App URL. Must be a valid URL.
+              {{command:--organization-id=<id>}}  Partner organization ID. Must be an existing organization.
+              {{command:--shop-domain=<domain>}}  Development store URL. Must be an existing development store.
+              {{command:--db=<system>}}           Database system. Must be one of: mysql, postgresql, sqlite3, oracle, frontbase, ibm_db, sqlserver, jdbcmysql, jdbcsqlite3, jdbcpostgresql, jdbc.
+              {{command:--rails-opts=<options>}}  Additional options. Must be a string containing one or more valid Rails options, separated by spaces.
```

Comment:
Not sure how hard these changes would be to make, or if it would break parsing from an underlying CLI framework?